src/Char.elm

```diff
@@ -16,25 +16,31 @@ module Char
 
 import Native.Char
 
+isBetween : Char -> Char -> Char -> Bool
+isBetween low high char = 
+    let code = toCode char
+    in  (code >= toCode low) && (code <= toCode high)
```

Comment:
I'd just import the functions in question: `import Basics ((&&), (>=), (<=))`

It is fine as long as the tests run. If there's a circular dependency, it'll just endlessly initialize modules until some piece of technology tells you to stop.